README.md

```diff
@@ -110,6 +110,13 @@ npm install @reactivex/rxjs@5.0.0-beta.1
 
 For CDN, you can use [npmcdn](https://npmcdn.com):
 
+5.0.0-beta.1 - 5.0.0-beta.11:
+
+https://npmcdn.com/@reactivex/rxjs/dist/global/Rx.umd.js
+
+
+5.0.0-beta.11 or higher:
+
 https://npmcdn.com/@reactivex/rxjs/dist/global/Rx.umd.js
```

Comment:
Actually, I believe this should be changed to `5.0.0-beta.12 or higher` need to point to `Rx.js` cause the previous section includes beta.11 (the current release) and correctly points to Rx.umd.js